### PR TITLE
CT-1038: Update subscription and time pass options

### PR DESCRIPTION
**Pull request by tomasstark**
This PR addresses [CT-1038](https://laterpay.atlassian.net/browse/CT-1038).

* It adds possibility to select `31` from the subscription/time pass length dropdown when "Day" is the period type.

### CODE REVIEW

File: v1/inc/classes/class-post-types.php [hunk 1]
```diff
@@ -352,32 +352,7 @@ public function get_post_tier( $post_content ) {
 	 * @return array array of options
 	 */
 	public static function get_duration_options() {
-		return [
```
Comment (tomasstark) [inline v1/inc/classes/class-post-types.php:355]: 😥

File: v1/inc/classes/class-post-types.php [hunk 2]
```diff
@@ -352,32 +352,7 @@ public function get_post_tier( $post_content ) {
...
-			17,
-			18,
-			19,
-			20,
-			21,
-			22,
-			23,
-			24,
-		];
+		return range( 1, 31 );
```
Comment (tomasstark) [inline v1/inc/classes/class-post-types.php:355]: ☺️

Comment (tomislavhren) [reply v1/inc/classes/class-post-types.php:355]: hardcore 😄

**Review by tomislavhren — APPROVED** [review]
🚀

**Pull request merged**